content/en/apps/guides/updates/preparing-for-4.md

```diff
@@ -150,6 +150,14 @@ This will give a list of every user and the version they're running as of the cu
 
 To upgrade app, run `npm update cht-conf`
 
+## cht-conf-test-harness
+
+The [cht-conf-test-harness](https://github.com/medic/cht-conf-test-harness) has [been upgraded](https://forum.communityhealthtoolkit.org/t/announcing-release-of-cht-conf-test-harness-3-0/2393) (in version 3.x of the cht-test-harness) to support testing CHT 4.x forms. When preparing to upgrade to CHT 4.x, it is important to use the [latest version](https://www.npmjs.com/package/cht-conf-test-harness?activeTab=versions) of the cht-conf-test-harness for automated tests.
```

Comment:
maybe link to docs instead of repo? http://docs.communityhealthtoolkit.org/cht-conf-test-harness/

Reply:
Good call!